lib/couchbase-orm/utilities/json_schema_loader.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+require 'couchbase-orm/json_schema'
+
+module CouchbaseOrm
+  module JsonSchemaLoader
+    include CouchbaseOrm::JsonSchema
+
+    class Error < RuntimeError; end
+
+    def config=(config)
+      CouchbaseOrm::JsonSchema::Loader.instance.initialize_schemas(ENV['CB_ORM_JSON_SCHEMA_PATH'])
```

Comment:
We should not define methods with a generic name (like `#config=`) in a module to be prepend without at least calling `super` in it.
And the  `instance.initialize_schema` make me think we should have used an initializer.

Reply:
I would initialize  the data and  store it in a struct available in the transcoder. I would be happy to pair with you on other solution :)